docs/it-manual/sre-playbook/disaster-recovery.md

```diff
@@ -0,0 +1,118 @@
+  {% endhint %}
+
+- Set POSTGRES_RESTORE_SNAPSHOT_IDENTIFIER to the snapshot to restore. 
+  
+  - To find the snapshot identifier go to the AWS console and navigate to RDS > Snapshots.
+  - Use the snapshot identifier (not the name, the identifier begins with `rds:`) for the value for `POSTGRES_RESTORE_SNAPSHOT_IDENTIFIER`.
+
+Do NOT change any other settings.
+
+{% hint style="danger" %}
+The `CIVIFORM_VERSION` MUST be the same version that was in use with the selected snapshot. Upgrading the version of CiviForm should NOT be done at the same time as restoring an environment.
+{% endhint %}
+
+## Initialize the deployment
+
+Verify that deployment tool is correctly setup by running `run bin/doctor --config=NEWCONFIGFILE`
+
+## Restore into new environment
+
+{% hint style="warning" %}
+This process will take some time to run. Depending upon the size of the database being restored expect on average 30-60 minutes for completion. There will be large periods where you need to wait while the setup runs, but there are also points where you will be prompted for values.
+{% endhint %}
+
+- To begin the restoration run `bin/setup --config=NEWCONFIGFILE`.
+- Wait as setup runs
+- When prompted enter the values for all the requested secrets.
+- When prompted for Postgres snapshot restore follow all prompts. You will be asked for the app prefix of the old environment. This is the value from `APP_PREFIX` of the original config file.
+- Wait as setup runs
+
+When complete, make note of the endpoint url listed at the end of the deployment. It will be the one that has the `APP_PREFIX` from your new config file and ends with "elb.amazonaws.com".
+
+At this point the new environment is setup and should be online at the above endpoint url if no errors occurred during this process.
+
+## Verify the new site is working
+
+Test that the site responds by using the endpoint url from the previous section.
+
+- Paste the endpoint url into your browser, you will see an invalid SSL certificate error and may not be able to process depending on your browser settings. If you can temporarily accept the invalid certificate you should see the CiviForm program page load.
+- Alternatively, you can run `curl --insecure --location PUT_ENDPOINT_URL_HERE`. The results should be HTML.
+
+## Change DNS settings to point to new site
+
+- Go to AWS Route53
+- Go to `Hosted zones`
+- Click the `Hosted zone name` link for the site you just restored
+- Select the `A` record type
+- Click "Edit record" in the flyout panel
+- Set Route traffic to
+    - Alias to Application and Classic Load Balancer
+    - The AWS Region you use
+    - Select the newly created environment's load balancer from the list
+- Click Save
+
+While is usually pretty instantaneous, it may take a few minutes for the DNS update to propagate.
```

Comment:
Yeah, this comment was specific to changing Route 53. I only know how the DNS is managed with Seattle. It would be good to update this with info that pertains to how others are running.